src/mmw/js/src/core/utils.js

```diff
@@ -607,6 +609,34 @@ var utils = {
             min: min,
             max: max
         };
+    },
+
+    // Parses version number from branch and git describe output
+    // Outputs version number and release notes URL
+    parseVersion: function(branch, gitDescribe) {
+        var version = 'Unknown',
+            url = RELEASE_NOTES_BASE_URL + '/latest';
+
+        if (branch === null && gitDescribe === null) {
+            version = 'Unknown';
+        } else if (branch === 'local' && gitDescribe === null) {
+            version = 'Local';
+        } else if (branch.startsWith('release')) {
+            version = branch.substr(branch.indexOf('/') + 1);
+            // Use this version for release notes
+            url = RELEASE_NOTES_BASE_URL + '/tag/' + version;
+        } else if (branch.startsWith('hotfix')) {
+            version = branch.substr(branch.indexOf('/') + 1);
+            // Use the original release notes
+            url = RELEASE_NOTES_BASE_URL + '/tag/' + version.match(MINOR_MAJOR_REGEX) + '0';
+        } else {
+            version = gitDescribe.substr(0, 18);
+        }
+
+        return {
```

Comment:
Stylistically, this return looks a little pythonic.  Consider dropping the quotes around the keys and converting to camelCase so the calling code looks more like a conventional JS object.

Reply:
Amended in 6d0b20e.